docs/code-standards-policy.rst

```diff
@@ -118,36 +171,418 @@ using the debops.apt_install_ role MUST be defined as (or similar):
+code of the role.
+
+.. _debops_policy__ref_code_standards_task_yaml_syntax:
+
+YAML syntax
+~~~~~~~~~~~
+
+Task definitions MUST use the native YAML syntax formatting. Ansible accepts
+various ways to define Ansible tasks. However, there are several advantages by
+agreening on the YAML syntax:
+
+- Unified coding style
+
+- Vertical formatting is easier to read
+
+- Supports complex parameter values (e.g. nested dictionaries)
+
+**Example:**
+
+Instead of ...
+
+.. code-block:: yaml
+
+   - name: Create plugin path
+     file: path={{ elasticsearch__path_plugins }} state=directory
+
+... use the correct YAML syntax:
+
+.. code-block:: yaml
+
+   - name: Create plugin path
+     file:
+       path: '{{ elasticsearch__path_plugins }}'
+       state: directory
+
+.. _debops_policy__ref_code_standards_task_disable_debug:
+
+Disable debug statements
+------------------------
+
+Role authors MUST NOT unconditionally use Ansible debug mechanisms such as the
+``debug`` module or the ``ignore_errors`` task statement in code which is used
+for normal operations. Released code is expected to be functional under every
+possible circumstance otherwise it is considered to be a bug which must be
+fixed on a best effort basis.
+
+For fragile or complex code paths it might be acceptable to to use the
+``debug`` statement with an increased ``verbosity`` level. This will only show
+the message, if :program:`ansible-playbook` is executed with one or more
+``--verbose`` options. For example:
+
+.. code-block:: yaml
+
+   - name: Show intermediate value from a lookup query
+     debug:
+       var: '{{ lookup("template", "lookups/fancy_lookup.j2") }}'
+       verbosity: 1
+
+
+.. _debops_policy__ref_code_standards_role_dependencies:
+
+Ansible role dependencies
+-------------------------
+
+.. _debops_policy__ref_code_standards_soft_role_dependencies:
+
+Soft role dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+Role dependencies are considered "soft" if they are defined in the ``roles``
+list of a playbook. This approach offers a higher flexibility as the user can
+choose which playbooks to run and which features to include.
+
+Whenever possible DebOps role authors MUST specify role dependencies via
+playbook instead of
+:ref:`"hard" dependencies <debops_policy__ref_code_standards_hard_role_dependencies>`
+in the :file:`meta/main.yml`.
+
+It's also possible to pass configuration values to a dependent role if the
+involved role is offering dedicated
+:ref:`dependent variables <debops_policy__ref_code_standards_dependent_variables>`
+for this purpose. With soft dependencies custom playbook authors are therefore
+free to pass values from arbitrary sources according to their requirements.
+
+**Example:**
+
+This is an example playbook for debops.slapd_ defining soft dependencies:
+
+.. code-block:: yaml
+
+    ---
+
+    - name: Manage OpenLDAP service
+      hosts: [ 'debops_service_slapd', 'debops_slapd' ]
```

Comment:
You should remove the `debops_slapd` version of the host group, the `debops_service_*` namespace is better.